bitnami/elasticsearch/templates/coordinating-statefulset.yaml

```diff
@@ -65,9 +65,14 @@ spec:
       topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.coordinating.topologySpreadConstraints "context" $) | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ template "elasticsearch.coordinating.serviceAccountName" . }}
-      {{- if .Values.coordinating.podSecurityContext.enabled }}
+      {{- if or .Values.coordinating.podSecurityContext.enabled .Values.coordinating.securityContext.enabled }}
       securityContext:
+        {{- if .Values.coordinating.securityContext.enabled }}
+        fsGroup: {{ .Values.coordinating.securityContext.fsGroup }}
+        {{- end }}
+        {{- if .Values.coordinating.podSecurityContext.enabled }}
         {{- omit .Values.coordinating.podSecurityContext "enabled" | toYaml | nindent 8 }}
+        {{- end }}
```

Comment:
I would make the `posdSecurityContext` to take precedence over the `securityContext`.
If not, it could duplicate fields.
I mean, if `podSecurityContext` is is set take it, if not, check the `securityContext`. While leaving the new `podSecurityContext` disabled by default, so it will continue working as in the past by default.
This applies to the rest of the files too.

Reply:
Right. 
What about this?
```
          {{- if or .Values.master.containerSecurityContext.enabled .Values.master.securityContext.enabled }}
          securityContext:
            {{- if .Values.master.containerSecurityContext.enabled }}
            {{- omit .Values.master.containerSecurityContext "enabled" | toYaml | nindent 12 }}
            {{ else }}
            runAsUser: {{ .Values.master.securityContext.runAsUser }}
            {{- end }}
          {{- end }}
         ```

Reply:
👍